docs/logging.md

```diff
@@ -20,7 +20,10 @@ We present how to set up a multi-node cluster, however, the same modifications c
    > **Beware:**
    > The following commands are meant to be run in a shell of type `bash` as mentioned in the [Quickstart guide][shell-type].
    > Running them with a different kind of shell interpreter can lead to undefined behaviour.
-   >
+   > 
+   > In addition, in order to simplify the debugging procedure of whole created cluster, `LogVerbosity` parameter in `configs/setup/system.json` can be set to different integers (e.g. 5).
```

Comment:
I think, it should be placed as usual paragraph before the notes in this section. Starting with the fact that kubelet has its own logs, that can have different verbosity. This verbosity can be controlled by this option. In addition it changes the sizes of per-container logs. And the default value.